components/shared/atoms/FormField/FormField.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Field, ErrorMessage } from 'formik';
+import styled from 'styled-components';
+
+import { Input } from 'components/shared/atoms/Input/Input';
```

Comment:
Could you please add index file for input (in order not to import `Input/Input`)